packages/components/src/navigation/NavTabList.scss

```diff
@@ -0,0 +1,166 @@
+@import '../../scss/custom.scss';
+
+$tab-height: 32px;
+$tab-drag-border-width: 1px;
+$tab-font-size: 1rem;
+
+$tab-link-side-padding: 24px;
+$tab-link-underline-spacing: 6px;
+
+$tab-link-hover-underline-color: $gray-400;
+
+$tab-link-active-color: $gray-200;
+$tab-link-active-underline-color: var(--dh-color-accent-bg);
+
+$tab-link-active-hover-underline-color: var(--dh-color-accent-bg);
+
+$tab-button-hover-color: $gray-200;
+$tab-button-separator-color: $gray-600;
+
+$tab-control-btn-offset: -8px;
+
+.nav-container {
+  display: flex;
+  flex-shrink: 0;
+
+  .nav-tabs {
+    --dragging-ant-color: $gray-300;
+
+    border: none;
+    height: $tab-height;
+    font-size: $tab-font-size;
+    flex-wrap: nowrap;
+    overflow-x: hidden;
+    position: relative;
+
+    &.dragging {
+      @include ants-base($gray-300, $background);
+    }
+
+    .btn-nav-tab {
+      // border: $tab-drag-border-width solid transparent;
```

Comment:
meant to be commented or deleted?

Reply:
I think deleted. Looks like the base style sheet includes this border for `.btn-link:not(.btn-link-icon)` already, so not sure if we need this here as it's redundant